Refactor Build container images page (#827)
nginx/agent: site/content/installation-upgrade/container-environments/docker-images.md

See the requirements and supported operating systems in the [NGINX Agent Technical Specifications]({{< relref "technical-specifications.md" >}}) topic.

## Deploy Offical NGINX and NGINX Plus Containers

Docker images are available in the [Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) NGINX documentation.

> The document provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.

Reviewer: ```suggestion
This guide provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```

See the requirements and supported operating systems in the [NGINX Agent Technical Specifications]({{< relref "technical-specifications.md" >}}) topic.

> ## Deploying NGINX and NGINX Plus on Docker
> 
> You can access container images by visiting the following page: 
> 
> [Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) 

Reviewer: ```suggestion
## Deploy NGINX and NGINX Plus on Docker

Docker images are available in the [Deploying NGINX and NGINX Plus on Docker](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/) NGINX documentation. 
```

> This page provides instructions on how to build images with the NGINX agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.

Reviewer: ```suggestion
The document provides instructions on how to build images with NGINX Agent and NGINX packaged together. It includes steps for downloading the necessary Docker images, configuring your Docker environment, and deploying NGINX and NGINX Plus containers.
```

Product names without ~the~ and capitalized.

   - Be sure to replace the example certificate and key filenames shown in the example command with your actual file names.
   - The file names in the *build/certs* directory must match those shown in the example.

## Run the NGINX Agent container

> Here is an example of how to run the NGINX Agent container using Docker:

Reviewer: ```suggestion
To run NGINX Agent container using Docker use the following command:
```

  nginx-agent
```

> To ensure that the REST Interface is correctly configured, you can curl the following endpoint from your terminal:

Reviewer: ```suggestion
To ensure that the REST Interface is correctly configured, you can use the `curl` command targeting the following endpoint from your terminal:

```

Leave a blank space between paragraphs and code blocks.

```shell
curl 0.0.0.0:8038/nginx/
```

> 

Reviewer: Should we also provide an example output or what to expect from this command to confirm the test?

> {{<bootstrap-table "table table-striped table-border">}}

Reviewer 2: One line 222 & 223 it should be:
```
- `make oss-image` builds an image containing NGINX Agent and NGINX open source.
- `make image` builds an image containing NGINX Agent and NGINX Plus.
```

Reviewer: updated

### Build NGINX open source images

> Run the following **make** command to build the default image, which uses Alpine as the base image.

Reviewer: ```suggestion
Run the following **make** command to build the default image, which uses Alpine as the base image:
```

```shell
IMAGE_BUILD_TARGET=install-agent-repo make oss-image
```

> To build an image with debian and older version of the NGINX Agent you can run the following command.

Reviewer: ```suggestion
To build an image with Debian and an older version of NGINX Agent you can run the following command:
```

Run the following `make` command to build the default image, which uses Ubuntu 24.04 (Noble) as the base image.

```shell
IMAGE_BUILD_TARGET=install-agent-repo make image
```

> To build an image with debian and older version of the NGINX Agent you can run the following command.

Reviewer: ```suggestion
To build an image with Debian and an older version of NGINX Agent you can run the following command:
```
